src/data_utils/ProcessTrafficData.py

```diff
@@ -195,6 +198,50 @@ def GenerateObsmat(traffic_data, data_path, save=True):
 
     return obsmat
 
+def GenerateCSV(traffic_data, data_path, save=True):
```

Comment:
Add this function to generate a csv. This csv can I then use for importing the obstacles into my package and publishing them.

Reply:
Looks good